api/indexer.oas2.json

```diff
@@ -982,6 +982,14 @@
             "$ref": "#/definitions/Asset"
           }
         },
+        "hashtype": {
```

Comment:
This enum should not be declared under `Account`, it is only required for `ProofParameters` and `ProofResponse`, these APIs are not currently supported by the indexer - only by Algod. 
Could you define it somewhere else so it'll still be generated but not under `Account`? It shouldn't be in the indexer to begin with, but it seems like the generator deletes the `enum` in the Java SDK if it's not defined here somewhere.